fdbserver/LogRouter.actor.cpp

```diff
@@ -256,6 +256,12 @@ ACTOR Future<Void> waitForVersion(LogRouterData* self, Version ver) {
 	// The only time the log router should allow a gap in versions larger than MAX_READ_TRANSACTION_LIFE_VERSIONS is
 	// when processing epoch end. Since one set of log routers is created per generation of transaction logs, the gap
 	// caused by epoch end will be within MAX_VERSIONS_IN_FLIGHT of the log routers start version.
+	TraceEvent("LogRouterWaitForVersion", self->dbgid)
```

Comment:
This is useful. However, I'm thinking a different approach: wrapping `waitForVersion` in an actor, that returns when `waitForVersion` returns and prints such a event every 60s.

Reply:
With that approach, we would not see the log unless `waitForVersion` returned. In the case I want to catch, it never returns

Reply:
Yes. This is exactly what I imagined!

Not sure if there are similar usage at other places. If so, we may create a template that takes a function as a parameter, something like `waitAndRunPeriodically(Future<T> f, double period, Func func)`. This is not needed for this PR, though.